app/src/main/java/com/cmput301f20t21/bookfriends/ui/library/OwnedListFragment.java

```diff
@@ -138,6 +138,13 @@ private void openAddEditActivity(@Nullable Book book) {
             startActivityForResult(intent, BOOK_ACTION.EDIT.getCode());
         }
     }
+    private void openDetailActivity(Book book){
+        Intent intent = new Intent(this.getActivity(), detailLibraryActivity.class);
+        intent.putExtra(BOOK_ACTION_KEY, BOOK_ACTION.VIEW);
```

Comment:
Don't think we need this here if the action after clicking on the book always goes to the details page